src/traces/scattermapbox/convert.js

```diff
@@ -89,7 +89,7 @@ module.exports = function convert(gd, calcTrace) {
         fill.layout.visibility = 'visible';
 
         Lib.extendFlat(fill.paint, {
-            'fill-color': trace.fillcolor,
+            'fill-color': trace.fillcolor
```

Comment:
There is still a number of refactoring changes in this PR that we would like to avoid for this implementation.
Please see the image below showing some (not all) of them.
![Screenshot from 2020-09-22 09-44-56](https://user-images.githubusercontent.com/33888540/93890935-dcfd6f00-fcb8-11ea-9722-ac159b365ba0.png)

You could view the rest using https://github.com/plotly/plotly.js/pull/5147/files

Reply:
I'll fix those changes tomorrow

Reply:
It's fixed now